.github/workflows/js_tests.yml

```diff
@@ -10,12 +10,6 @@ jobs:
   test:
     runs-on: ubuntu-latest
     timeout-minutes: 20
-    strategy:
-      fail-fast: false
-      matrix:
-        node-version:
-          - 12
```

Comment:
For what it's worth, all our production builds are built with node 12. Perhaps spelling should also have an `if ... == 14`?

Reply:
Lets switch tests to node 12 then? And when production shift to 14 we will follow

Reply:
I think it's easier to keep tests running for both so you are already compatible when you do switch.

Reply:
looks like it takes ~8 minutes for each of these GH tests,
and I didn't encounter any differences in their results,
so if we will remove one of them we can save many hours eventually.

Reply:
IMHO removing one means removing support. So if we remove Node 14 from our CI, we're effectively saying that using Node 14 isn't supported. Is that what we want?

Reply:
Do you know when production is moving to node 14?